### PR TITLE
Added new docs for v3_api

### PR DESCRIPTION
**Pull request by GustavoCaso**
@kytrinyx I hope you mean updating the docs by this

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+* Start Xcode create a new project.
+* Select iOS-->Application and then Single View Application and click Next.
+* Enter `Bob` as your Product Name, set the language to Swift and then click Next.
+* Accept the default settings for the remaining screens and finish creating your project.
+* You will see two targets in your project, one for the application named `Bob` and one for tests named `BobTests`. If you chose to create an OSX application instead of an iOS application, you will not see a `BobTests` target and will have to create it manually.
+* In the `BobTests` target, open the file named `BobTests.swift` replace the contents with the test file you got from exercism.
+* Add a new file to the `Bob` target folder and name it `Bob.swift`.
+* Select `Bob.swift` and in the File Inspector make sure that both `Bob` and `BobTests` are checked in the Target Membership area.
+* Now use CMD-U to run the tests, and happy coding!
+
```
Comment (kytrinyx) [inline docs/RESOURCES.md]: I think this whole section should go in the TESTS.md, since you can't run the tests until this has been done.

**Comment by kytrinyx** [comment]
Yepp, this is what I meant :)

**Comment by GustavoCaso** [comment]
Glad to hear, Is a boring task but is a must do

**Comment by masters3d** [comment]
Thank you Gustavo! 😊 Nice work!

**Comment by kytrinyx** [comment]
Yeah, I'm so excited to be able to get rid of the `help` site.

**Comment by GustavoCaso** [comment]
Yes sounds really rational to get rid of it

**Pull request merged**